From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Sat, 8 Dec 2012 12:45:33 +0100
Subject: [PATCH] HACK: move LDFLAGS behind objects

LDFLAGS is used for '-ldl' only, which must be specified after any object
files.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 mk/Compile.mk |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mk/Compile.mk b/mk/Compile.mk
index 7771ef0..abad750 100644
--- a/mk/Compile.mk
+++ b/mk/Compile.mk
@@ -24,7 +24,7 @@ compile_c	= $(CC) $(CFLAGS) $(CPPFLAGS) -Wp,-MD,$(tmpdep) -c -o $@ $<
 compile_cc	= $(CXX) $(CXXFLAGS) $(CPPFLAGS) -Wp,-MD,$(tmpdep) -c -o $@ $<
 fixup_deps	= sed -e "s|.*\.o:|$@:|" < $(tmpdep) > $(depfile) && rm -f $(tmpdep)
 
-link_app	= $(CC) $(LDFLAGS) -o $@  $^ $(LDLIBS)
+link_app	= $(CC) -o $@  $^ $(LDFLAGS) $(LDLIBS)
 link_so		= $(CC) $(LDFLAGS) -shared -Wl,-soname,$(@F) -o $@ $^ $(LDLIBS)
 ar_lib		= rm -f $@ && ar -r $@ $^ && ranlib $@
 
